src/modules/dashboard/components/SmiteDialog/SmiteDialogForm.tsx

```diff
@@ -0,0 +1,199 @@
+import React, { useMemo, useCallback, ReactNode } from 'react';
+import { FormikProps } from 'formik';
+import { defineMessages } from 'react-intl';
+import sortBy from 'lodash/sortBy';
+
+import Button from '~core/Button';
+import { ItemDataType } from '~core/OmniPicker';
+import { ActionDialogProps } from '~core/Dialog';
+import DialogSection from '~core/Dialog/DialogSection';
+import { Select, Input, Annotations, SelectOption } from '~core/Fields';
+import Heading from '~core/Heading';
+import SingleUserPicker, { filterUserSelection } from '~core/SingleUserPicker';
+
+import { Address } from '~types/index';
+import HookedUserAvatar from '~users/HookedUserAvatar';
+import { AnyUser, OneDomain } from '~data/index';
+
+import { FormValues } from './SmiteDialog';
+import TeamDropdownItem from './TeamDropdownItem';
+
+import styles from './SmiteDialogForm.css';
+
+const MSG = defineMessages({
+  title: {
+    id: 'dashboard.SmiteDialog.SmiteDialogForm.title',
+    defaultMessage: 'Smite',
+  },
+  team: {
+    id: 'dashboard.SmiteDialog.SmiteDialogForm.team',
+    defaultMessage: 'Select Team in which punishment should happen',
+  },
+  who: {
+    id: 'dashboard.SmiteDialog.SmiteDialogForm.who',
+    defaultMessage: 'Pick who is the target',
```

Comment:
I know that this is the copy from the specs but
1. it doesn't sound very good & grammatically I think it's incorrect - it should be at least `Pick who the target is`
2. it doesn't sound good - too wordy. I ran it by my native speaker friend and he suggested `Select a target` (preferable option) or `Choose a target`.
So I would suggest to change it but it's up to you...